### ISSUE TITLE
LGAL Approach - Cannot comply with altitude restrictions

### ISSUE THREAD

**Issue by albar965** [albar965/littlenavmap#379]
IFR Thessaloniki Makedonia (LGTS) to Alexandroupolis Dimokritos (LGAL).pln

Thessaloniki Makedonia (LGTS) 10 to Alexandroupolis Dimokritos (LGAL)
Via ALP and NDB FN07 (N07) to runway 07 .

> Cannot comply with altitude restrictions.

```
APPCH:020,N,N07, ,FN07,LG,P,C,E CF, ,   ,IF, ,ALP,LG,D,B,      ,    ,    ,    ,    ,+,02000,     ,     , ,   ,    ,   ,ALP,LG,D,B, ,0, ,S;
APPCH:030,N,N07, ,EN07,LG,P,C,EY E, ,   ,CF, ,ALP,LG,D,B,      ,    ,    ,0530,0061, ,00060,     ,     , ,   ,-300,   , , , , , ,0, ,S;
APPCH:040,N,N07, ,ALP,LG,D,B,EY M, ,   ,CF, ,ALP,LG,D,B,      ,    ,    ,0530,0001,+,00430,     ,     , ,   ,    ,   , , , , , ,0, ,S;
APPCH:050,N,N07, , , , , ,  M ,R,   ,CA,Y, , , , ,      ,    ,    ,2330,    ,+,02500,     ,     , ,   ,    ,   , , , , , ,0, ,S;
APPCH:060,N,N07, ,ALP,LG,D,B,E   ,R,   ,DF, , , , , ,      ,    ,    ,    ,    ,+,04000,     ,     , ,   ,    ,   , , , , , ,0, ,S;
APPCH:070,N,N07, ,ALP,LG,D,B,EE H,R,   ,HM, , , , , ,      ,    ,    ,2080,T010, ,     ,     ,     , ,   ,    ,   , , , , , ,0, ,S;
```

**Comment by martinlest** [comment]
V2.2.4 - 'Cannot Comply With Altitude Restrictions'... this comes up only after I start adding SID/STAR?

**Comment by martinlest** [comment]
V2.2.4 - 'Cannot Comply With Altitude Restrictions'... this comes up only after I start adding SID/STAR? (Happens fevery now & again, not specific to LGTS to LGAL)

**Comment by albar965** [comment]
@martinlest : Please attach examples or comments here if you run into this. It can happen while building a flight plan or if the cruise altitude is too low. But it should not happen for an apparently OK flight plan.
Alex

**Comment by martinlest** [comment]
Hi Alex... Current example: ZGGG to VHHH. Only waypoint en route is the VORDME 'NLG' (Nanlang). Cruise alt = 18000'. No problems in LNM. As soon as I add the SID SAR81D for Rwy02L departure though, I get 'Cannot comply with Altitude Restriction'. Remove the SID and the warning goes away. As far as I remember, I only see this warning when I add a SID or STAR... (not every time!). If you need more detail, please let me know. Thanks.

**Comment by albar965** [comment]
ZGGG to VHHH is working OK here. Usually when LGTS to LGAL is fixed it will fix others too.
Alex

**Comment by martinlest** [comment]
Hi Alex,

 

“Usually when LGTS to LGAL is fixed it will fix others too.”

 

You mean if I fix one example, all such errors will then be fixed? I get this quite often these days (never used to).. today LOWW to EHAM. All is fine until I add SID/STAR, then I get the ‘Cannot Comply with Altitude Restrictions’ error. 

 

What do I need to do to ‘fix’ this (and then see if all are fixed)? Sorry if this has been explained somewhere already...

 

Regards,

 

Martin

**Comment by albar965** [comment]
@martinlest : If you know other cases, tell me. I need the whole route description to reproduce it.
What I fixed should not happen with SID and STAR. Only approaches.
Alex

**Comment by martinlest** [comment]
Hi Alex,

 

Well I am getting this problem very regularly now, half my flights perhaps (I usually do assign an ILS approach and SID/STAR). 

 

I don’t know if this is helpful to you, but I just loaded up an XP11 format flight plan, containing an ILS approach (and SID/STAR) and got the ‘can’t comply..’ notice. I also have the same flight plan in ‘FMS 3’ format (needed for the a/c FMC). I loaded that into LNM instead.. no error as expected (I had to restart LNM though for the error to disappear). I then went back and loaded the XP11 format file into LNM again, expecting the ‘can’t comply...’ error to reappear, but it didn’t. I attach the XP11 flight plan anyway. If that plan doesn’t reproduce the error for you, let me know and I’ll email you more.

 

Thanks,

 

Martin
